src/test/suite/install.test.ts

```diff
@@ -16,7 +16,7 @@
  */
 'use strict';
 
-import * as assert from 'assert';
+import {assert} from 'chai';
```

Comment:
it provides more functionality and avoids usage of deprecated "assert.equal" method